app/src/main/java/com/framgia/forder/screen/shopmanagement/ShopManagementFragment.java

```diff
@@ -8,6 +8,7 @@
 import android.view.View;
 import android.view.ViewGroup;
 import com.framgia.forder.R;
+import com.framgia.forder.data.model.ShopManagement;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.model.ShopManagement. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports